system/modules/comments/classes/Comments.php

```diff
@@ -603,7 +618,7 @@ public static function notifyCommentsSubscribers(\CommentsModel $objComment)
 
 		while ($objNotify->next())
 		{
-			// Don't notify the commentor about his own comment
+			/** @var \CommentsNotifyModel $objNotify */
```

Comment:
This should not be necessary if `\CommentsNotifyModel::findActiveBySourceAndParent` has a correct docblock?